calcom/cal.com
# Monorepo/app store MS Teams Integration

## What does this PR do?

This PR adds the option to create a MS Teams link as a event location. Note that this only works with enterprise accounts (ie. work or school)



Addresses #1462

## Type of change




## How should this be tested?



Setup
- Ensure that you created MS Graph credentials in your .env file
- In Azure -> App Registration -> "Your app" -> API Permissions, add the `
OnlineMeetings.ReadWrite` permission 
- Enroll in the MS Developer program to gain access to a test enterprise account
    - Link: https://developer.microsoft.com/en-us/microsoft-365/dev-program
    - I can also provide the account that I used

Testing
 - Create a new user 
 - Once logged in go to the url `http://localhost:3000/apps/installed`
 - Under "Conferencing" connect to MS Teams
 - On the MS sign in page enter the test enterprise account details
 - Once connected create a new event type and set the location to MS Teams
 - With the event type, create a new booking. You should see that the meeting URL is created

## Checklist
